src/cloud_provider/digitalocean/common.rs

```diff
@@ -54,7 +79,11 @@ struct Cluster {
 }
 */
 
-pub fn get_uuid_of_cluster(token: &str, kubeID: &str) -> Result<String, SimpleError> {
+// retreive the digital ocean uuid of the kube cluster from our cluster name
+pub fn get_uuid_of_cluster_from_name(
```

Comment:
I can see in the code a comment like "retrieve the cluster uuid, useful to link DO registry to k8s cluster". Can you please add some comments to explain why we need to have it. Isn't it possible to use the registry without the ID of the container?